flow/flow.go

```diff
@@ -94,10 +94,18 @@ type RawPackets struct {
 	RawPackets []*RawPacket
 }
 
+type LayerKeyMode int
+
+const (
+	L2L3KeyMode LayerKeyMode = 0 // uses Layer2 and Layer3 for hash computation, default mode
+	L3KeyMode   LayerKeyMode = 1 // uses Layer3 only and layer2 if no Layer3
```

Comment:
consider chaining the name of the variable to L3PreferredKeyMode just to make it super clear that we still use L2 if L3 doesn't exist

Reply:
I'll do, thx for the suggestion